test/fixtures/cross_agent_tests/distributed_tracing/trace_context.json

```diff
@@ -289,6 +289,33 @@
       }
     }
   },
+  {
```

Comment:
This isn't part of the cross-agent tests right now, but I think I'll open a PR for it.